src/utils.jl

```diff
@@ -52,11 +52,436 @@ function uniquenames(names, newnames)
 end
 
 function projjsonstring(code; multiline=false)
-  spref = spatialref(code)
-  wktptr = Ref{Cstring}()
-  options = ["MULTILINE=$(multiline ? "YES" : "NO")"]
-  GDAL.osrexporttoprojjson(spref, wktptr, options)
-  unsafe_string(wktptr[])
+  wkt2str = CoordRefSystems.wkt2(code)
+  wkt2_to_projjson(wkt2str, multiline=multiline)
+end
+
+# Constants for WKT2 to PROJJSON conversion
+const DEG_TO_RAD = π / 180
+
+function wkt2_to_projjson(wkt2str; multiline=false)
+  # For now, let's implement a basic parser that handles common CRS types
+  # First identify the type of CRS
+  if startswith(wkt2str, "GEOGCRS")
+    return parse_geogcrs(wkt2str, multiline=multiline)
```

Comment:
Please remove `return` from the last statements of functions. Julia automatically returns the last statement.